core/include/gnuradio-4.0/Block.hpp

```diff
@@ -544,7 +585,39 @@ struct Block : protected std::tuple<Arguments...> {
         }
     }
 
-    Block(Block &&other) noexcept : std::tuple<Arguments...>(std::move(other)), _settings(std::move(other._settings)) {}
+    Block(Block &&other) noexcept
+        : std::tuple<Arguments...>(std::move(other))
+        , _settings(std::move(other._settings))
+        , numerator(std::move(other.numerator))
+        , denominator(std::move(other.denominator))
+        , stride(std::move(other.stride))
+        , stride_counter(std::move(other.stride_counter))
+        // , state(std::move(other.state)) // atomic, not included in move operations
+        , msgIn(std::move(other.msgIn))
+        , msgOut(std::move(other.msgOut))
+        , ports_status(std::move(other.ports_status))
+        , _output_tags_changed(std::move(other._output_tags_changed))
+        , _mergedInputTag(std::move(other._mergedInputTag)) {}
+
+    Block &
+    operator=(Block &&other) noexcept {
+        auto tmp = std::move(other);
```

Comment:
When I actually try to move-assign a graph, compilation fails in several places, including Port::operator=(Port &&other) (members wrongly prefixed with `_`), and below.